source/views/Extension/Views/ImportWallet/styles.js

```diff
@@ -0,0 +1,110 @@
+import { makeStyles } from '@material-ui/core/styles';
+import zIndex from '@material-ui/core/styles/zIndex';
+
+export default makeStyles((theme) => ({
+  dragDropContainer: {
+    border: '2px dashed #D1D5DB',
+    boxShadow: '0px 1px 2px rgba(0, 0, 0, 0.05)',
+    height: '142px',
+    width: '370px',
+    borderRadius: 6,
+    marginBottom: 20,
+  },
+  dragDropContainerError: {
+    border: '2px dashed #E53C3C',
+    boxShadow: '0px 1px 2px rgba(0, 0, 0, 0.05)',
+    height: '142px',
+    width: '370px',
+    borderRadius: 6,
+    marginBottom: 40,
```

Comment:
there's a lot of repeated styles here, you could abstract them to a common constant like
```js
  const dragDropBase = {
    boxShadow: '0px 1px 2px rgba(0, 0, 0, 0.05)',
    height: '142px',
    width: '370px',
    borderRadius: 6,
  };
  
  ...
  export default makeStyles((theme) => ({
  dragDropContainer: {
    ...dradDropBase,
    border: '2px dashed #D1D5DB',
    marginBottom: 20,
  },
  dragDropContainerError: {
    ...dragDropBase,
    border: '2px dashed #E53C3C',
    marginBottom: 40,
    }

Reply:
or even just use `dragDropContainer` as the base and overwrite whatever is needed